apps/vr-tests/src/stories/SliderNext.stories.tsx

```diff
@@ -1,11 +1,9 @@
-/*! Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. */
```

Comment:
We should probably delete these line from all vr tests 😆.

Reply:
Pretty sure it's universally unnecessary, but I'd rather not add even more noise to this PR.